binding/python/examples/theano/cnn.py

```diff
@@ -71,7 +71,7 @@ def momentum(cost, params, learning_rate, momentum):
     updates = []
 
     for p, g in zip(params, grads):
-        mparam_i = theano.shared(np.zeros(p.get_value().shape, dtype=theano.config.floatX))
+        mparam_i = mv_shared(np.zeros(p.get_value().shape, dtype=theano.config.floatX))
```

Comment:
Actually I suggested we could do something like 'theano.shared = mv_shared' in somewhere after the import of theano so that people don't need to modify their own code when try to use shared variable.